src/platforms/ruby/common/configuration/options.mdx

```diff
@@ -52,10 +52,16 @@ end
 
 `before_send`
 
-: Provide a lambda or proc. This will be `called` before sending an event to Sentry. Receives an `event` and `hint` as parameter. `hint` is a dict `{:exception => ex | nil, :message => message | nil}`. It is possible to mutate the event, also if this function returns `nil` the event will be dropped and not sent.
+: Provide a lambda or proc. This will be `called` before sending an error event to Sentry. Receives an `event` and `hint` as parameter. `hint` is a dict `{:exception => ex | nil, :message => message | nil}`. It is possible to mutate the event, also if this function returns `nil` the event will be dropped and not sent.
 
 <PlatformContent includePath="configuration/before-send" />
 
+`before_send_transaction`
+
+: Provide a lambda or proc. This will be `called` before sending a transaction event to Sentry. Receives an `event` and `hint` as parameter. `hint` is currently empty but might be augmented in the future. It is possible to mutate the event, also if this function returns `nil` the event will be dropped and not sent.
+
+<PlatformContent includePath="configuration/before-send-transaction" />
```

Comment:
```suggestion

: Provide a lambda or proc. This will be `called` before sending a transaction event to Sentry. It receives an `event` and `hint` as a parameter. The `hint` event is currently empty, but might be augmented in the future. It's possible to mutate the event. Also, if this function returns `nil`, the event will be dropped and not sent.

<PlatformContent includePath="configuration/before-send-transaction" />
```
I wonder if it's possible to add a bit of additional context here and also for the "before_send_transaction" the same way we have additional context for "Background_worker_threads" and "Backtrace_cleanup_callback".  It's really helpful to talk about why a user might want to do the things we're describing.